src/midiEditorCommands.cpp

```diff
@@ -2292,6 +2293,96 @@ void postToggleFunctionKeysAsStepInput(int command) {
 	}
 }
 
+void postMidiToggleMute(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int eventCount = countSelectedEvents(take);
```

Comment:
Sorry I didn't spot this in the first pass, but I just realised countSelectedEvents is an OSARA function which has to walk all the selected events itself. We then go and walk them again when we get selected notes and CCs. That seems like redundant walk and could be slower than it needs to be if there are a lot of selected events. I suggest you just get the selected notes and CCs (since you're going to do that anyway) and add the counts together. Unless there's another specific reason you did it this way?